tests/utils/host_utils.cpp

```diff
@@ -655,6 +656,11 @@ void get_gridsize_from_env(int *const dims)
   }
 }
 
+void get_gridsize_from_env(int *const dims)
```

Comment:
I think we can just get rid of this wrapper function and call `get_size_from_env(dims, "QUDA_TEST_GRID_SIZE");` directly